docker/bin/upgrade-ubuntu.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
These are very similar. Have you considered making the platform a parameter?

Reply:
I ended up making this `upgrade.sh` and `versions.sh`, not a big deal to leave the platform name out of the script name actually.